sdk/cosmosdb/cosmos/test/internal/session.spec.ts

```diff
@@ -120,9 +124,9 @@ describe.skip("Session Token", function (this: Suite) {
     const container2 = clientB.database(dbId).container(containerId);
     await Promise.all([createItem(container), createItem(container2)]);
 
-    // Create an item using client
-    const id = await createItem(container);
-    const { resource, statusCode } = await container2.item(id).read();
+    // Container read with consistency level and max integrated cache staleness(milliseconds)
+    const container3 = clientC.database(dbId).container(containerId);
```

Comment:
Can you please explain on why new client and container is used? 
And also if its new intentional separation, then won't existing read needs to be still around?